upstream-community-operators/postgres-operator/postgres-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,277 @@
+      mediatype: image/png
+  links:
+  - name: Documentation
+    url: https://postgres-operator.readthedocs.io/en/latest/
+  - name: GitHub
+    url: https://github.com/zalando/postgres-operator
+  customresourcedefinitions:
+    owned:
+    - name: postgresqls.acid.zalan.do
+      version: v1
+      kind: postgresql
+      displayName: Postgres-Operator
+      description: Creates a PostgreSQL cluster managed by Postgres-Operator.
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: true
+  - type: AllNamespaces
+    supported: true
```

Comment:
Does this operator watch CRs created in namespaces outside of the operator's deployed namespace?

Reply:
Yes and no. [AllNamespaces](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#specify-the-namespace-to-watch) is possible, but one can not specify certain namespaces to watch. Updated the parameters